examples/packages/WORKSPACE

```diff
@@ -26,3 +26,8 @@ node_repositories(package_json = ["//:package.json"])
 load(":setup_workspace.bzl", "packages_example_setup_workspace")
 
 packages_example_setup_workspace()
+
+load("@build_bazel_rules_nodejs//:package.bzl", "check_rules_nodejs_version")
```

Comment:
users should load it through `//:defs.bzl` so they don't need two load statements in their WORKSPACE

Reply:
👍